Volume/Views/MainView/ConnectionFailedView.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  NoConnectionView.swift
+//  Volume
+//
+//  Created by Sergio Diaz on 4/10/21.
+//  Copyright © 2021 Cornell AppDev. All rights reserved.
+//
+
+import SwiftUI
+
+struct ConnectionFailedView: View {
+    var body: some View {
+        VStack(spacing: 0) {
+            Spacer()
+
+            Image("no-connection")
+                .resizable()
+                .aspectRatio(contentMode: .fit)
+                .frame(width: 58, height: 58)
+                .padding(.bottom, 20)
+
+            Text("No Connection")
+                .font(.begumMedium(size: 23))
+                .padding(.bottom, 2)
+
+            Text("Please try again later")
+                .font(.system(size: 17))
```

Comment:
I'm guessing this should be hevletica or one of the volume-specific fonts (would need to check figma), but could be wrong

Reply:
I'm actually not sure why but Maggie's design actually include the default font within them.  No clue why, but I think it looks pretty good.